nixos/tests/samba.nix

```diff
@@ -5,34 +5,52 @@ import ./make-test-python.nix ({ pkgs, ... }:
 
   meta.maintainers = [ pkgs.lib.maintainers.eelco ];
 
-  nodes =
-    { client =
-        { pkgs, ... }:
-        { fileSystems = pkgs.lib.mkVMOverride
-            { "/public" = {
-                fsType = "cifs";
-                device = "//server/public";
-                options = [ "guest" ];
-              };
-            };
+  nodes = {
+    client = { pkgs, ... }: {
+      services.samba-wsdd = {
+        enable = true;
+        interface = "eth1";
+        workgroup = "WORKGROUP";
+        hostname = "CLIENT-WSDD";
+        discovery = true;
+        extraOptions = [ "--verbose" "--no-host" ];
```

Comment:
Let's drop this line, it should still work.

Reply:
If this drop line - the test freezing. Needed options `"--no-host"`